src/app/stacktrace_win.h

```diff
@@ -108,6 +112,66 @@ BOOL CALLBACK straceWin::EnumModulesCB(LPCSTR ModuleName, DWORD64 BaseOfDll, PVO
 }
 
 
+/**
+* Cuts off leading 'dir' path from 'file' path, otherwise leaves it unchanged
+* returns true iff 'dir' is an ancestor of 'file'
```

Comment:
'iff' usually means 'if and only if'. Anyway, I replaced the abbreviation to avoid any confusion